tests/utapi/awsNodeSdk.js

```diff
@@ -471,4 +471,26 @@ describe('utapi v2 metrics incoming and outgoing bytes', function t() {
             next => deleteBucket(bucket, next),
         ], done);
     });
+
+    it('should not push a metric for a filtered bucket', done => {
+        const bucket = 'utapi-event-filter-deny-bucket';
+        const objectSize = 1024 * 1024;
+        const objSize = objectSize * 2;
+        const key = '1.txt';
+        async.series([
+            next => createBucket(bucket, next),
+            next => putObject(bucket, key, objSize, next),
+            next => wait(WAIT_MS, next),
+            next => wait(WAIT_MS, () => {
```

Comment:
```suggestion
            next => wait(2 * WAIT_MS, () => {
```
or (if WAIT_MS is enough)

```suggestion
            next => wait(WAIT_MS, () => {
```